### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by henkvantijen** [JesseCorrington/bitcoinj-wallet-export#1]
Hello Jesse ;)
A friend asked to check if his Hive wallet can be salvaged. Tried your tool, but get build error as per below. Any hints for setting the correct build parameters (2 years later :) ? 

henk$ ./build.sh -e
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for wallet.export:bitcoinj-wallet-export:jar:0.1
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ line 32, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ----------------< wallet.export:bitcoinj-wallet-export >----------------
[INFO] Building bitcoinj-wallet-export 0.1
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ bitcoinj-wallet-export ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /Users/henq/Downloads/bitcoinj-wallet-export-master/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ bitcoinj-wallet-export ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 2 source files to /Users/henq/Downloads/bitcoinj-wallet-export-master/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 5 is no longer supported. Use 7 or later.
[ERROR] Target option 5 is no longer supported. Use 7 or later.
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.657 s
[INFO] Finished at: 2019-05-12T12:18:20+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project bitcoinj-wallet-export: Compilation failure: Compilation failure: 
[ERROR] Source option 5 is no longer supported. Use 7 or later.
[ERROR] Target option 5 is no longer supported. Use 7 or later.
[ERROR] -> [Help 1]

**Comment by JesseCorrington** [comment]
Sorry, I haven't checked this repo in a long time. Are you still trying to salvage the wallet? What platform did you run the build on?

**Comment by philtubman** [comment]
Hi, I am getting the same error. Running on Big Sur. Any help appreciated. Trying to salvage some residual coinage from old Hive wallet file....

**Comment by henkvantijen** [comment]
Yes,  residual coinage can be worth a fine Porsche, some day LOL !

**Comment by philtubman** [comment]
Did you work out how to unlock that hive wallet in the end @henkvantijen

**Comment by henkvantijen** [comment]
No #sosad

**Comment by kevinm90** [comment]
I got it to work on MacOS Catalina (10.15.7) and sapmachine 11 as JDK by adding these lines in the pom:
```
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-compiler-plugin</artifactId>
    <configuration>
        <source>1.7</source>
        <target>1.7</target>
    </configuration>
</plugin>
```

Also created a PR: https://github.com/JesseCorrington/bitcoinj-wallet-export/pull/2

**Comment by JesseCorrington** [comment]
@henkvantijen, @philtubman, @kevinm90: Were any of you able to recover your old wallets? I didn't realize my primary GitHub email was set to an old work email I no longer have, so I wasn't getting notified of comments on this repo. I'm around now if you want further assistance.

**Comment by philtubman** [comment]
Hi Jesse - no i didn't recover the old wallet yet. There isn't really
enough residual in it to bother financially, but i like tinkering with it
to see if I can. I'm on MacOS 11.1 right now and can see above that kevin
seems to have worked it out. Is that a good solution for you or would you
do something else to recover it?
Phil

**Comment by philtubman** [comment]
Hey - so close to getting this to work. Have built it on the new branch and have tried the ./run command with my password and it just says quote>   any idea what i need to do.

**Comment by philtubman** [comment]
sorry dude - just got it - had to put "" around my password!!

**Comment by JesseCorrington** [comment]
Awesome, glad you got it working @philtubman.

**Comment by luelista** [comment]
Just wanted to let you know and thank you for this software 😅

For me it worked like a charm, just running build.sh and run.sh on macOS 11.6.1 with mvn installed from Homebrew. It recovered my old bitcoin wallet with coins worth $6.50 in it :D

**Comment by JesseCorrington** [comment]
@luelista, Glad to hear it worked for you! Too bad it wasn't worth more.
